packages/hub/src/scope.ts

```diff
@@ -75,6 +75,12 @@ export class Scope implements ScopeInterface {
   /** Request Mode Session Status */
   protected _requestSession?: RequestSession;
 
+  /**
+   * A place to stash data which is needed at some point in the SDK's event processing pipeline but which shouldn't get
+   * sent to Sentry
+   */
+  protected _processingMetadata?: { [key: string]: any } = {};
```

Comment:
Why not use `Record`?
```suggestion
  protected _processingMetadata?: Record<string, any> = {};
```

Reply:
¯\\_(ツ)_/¯ Mostly habit.

Is there an advantage you see to using `Record`?

Reply:
Not really, is just a nit about using TS utility types :D.